pages/about.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link href="styles/about.css" rel="stylesheet">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>&hearts;shell-bee&hearts;</title>
 </head>
 <body>
+  <header>
+    <h1>&hearts;about&hearts;</h1>
+    <nav>
+      <div class="dropdown">
+          <span>navigation:</span>
+          <div class="dropdown-content">
+              <a href="index.html" >home</a>
+              <br><a href="about.html">about</a>
+              <br><a href="portfolio.html">portfolio</a>
+            </div>
+        </div>
+      </nav>
+      <br>
+  </header>
+  <main>
+      <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/meish.png" alt="Photo of myself.">
+
+    <p>
+      <h2>Shelby Rosel Faulconer</h2>
+      <div id="about-me">
+        <h3>&hearts;&hearts;&hearts;<br>
+          <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/biju.gif" alt="gif of Biju from Hamtaro">
+        </h3>
+        My name is Shelby (shell + bee). I study full-stack development and have a rich background in biology, mathematics, and chemistry.
+        Expressing myself is easiest through visual mediums and writing music.
+        <br>My passions extend deeply and I am a forever student.
+        <h3>
+            <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/theythem.webp" alt="Moving image of license plate with pronouns they/them" width=150px>
+        <br>&hearts;&hearts;&hearts;</h3>
+      </div>
+    </p>
+  </main>
+
+<footer>
+  Copyright &copy; Shelby R. Faulconer, 2022
+</footer>
```

Comment:
Since the `footer` is inside the `body` element, we would want to indent the `footer` so we can easily see that relationship at a glance.